frontend/src/components/BalancePresentation.js

```diff
@@ -50,7 +49,9 @@ class BalancePresentation extends React.Component {
             <p>{balanceLabel}</p>
           </Col>
           <Col className="pl-0 d-flex justify-content-end">
-            <p>{this.state.balance} €</p>
+            <p>
+              {this.props.salary - this.state.saving - this.state.expenses}€
```

Comment:
Actually, now that the savings feature was integrated, this should be:
```js
{this.props.salary - this.props.saving - this.state.expenses}€
```
And if the feature for showing expense on landing page gets integrated before this one, this would be
```js
{this.props.salary - this.props.saving - this.props.expenses}€
```

Reply:
done